src/store/index.ts

```
@@ -319,6 +322,7 @@ export const useBikeTagStore = defineStore(BikeTagDefaults.store, {
       })
     },
     fetchTags(cached = true) {
+      debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)